index.html

```diff
@@ -0,0 +1,287 @@
+            </div>
+            <div class="language">
+              <!-- card Language  -->
+              <ul>
+                <li class="a l-a">html</li>
+                <li class="b">css</li>
+                <li class="c">javascript</li>
+              </ul>
+            </div>
+            <div class="button">
+              <!-- card button  -->
+              <button>See project</button>
+            </div>
+          </div>
+        </a>
+        </div>
+        <!-- //card 2 -->
+        <div class="card">
+            <a href="">
+          <div class="img">
+            <!-- Card Image  -->
+            <img src="./Pictures/Snapshoot Portfolio 2.png" alt="" />
+          </div>
+          <div class="card-main">
+            <div class="card-name-h1">
+              <!-- Card main heading -->
+              <h1>Multi-Post Stories</h1>
+            </div>
+            <div class="experience">
+              <ul>
+                <!-- card Frame -->
+                <li class="a">CANOPY</li>
+                <li class="b">Back End Dev</li>
+                <li class="c">2015</li>
+              </ul>
+            </div>
+            <div class="parga">
+              <!-- Card Des -->
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="language">
+              <ul>
+                <!-- Card Language -->
+                <li class="a l-a">html</li>
+                <li class="b">css</li>
+                <li class="c">javascript</li>
+              </ul>
+            </div>
+            <div class="button">
+              <!-- Card Button  -->
+              <button>See project</button>
+            </div>
+          </div>
+        </a>
+        </div>
+        <!-- //card 3 -->
+        <div class="card">
+            <a href="">
+          <div class="img">
+            <!-- Card Image  -->
+            <img src="./Pictures/Snapshoot Portfolio 3.png" alt="" />
+          </div>
+          <div class="card-main">
+            <div class="card-name-h1">
+              <!-- Card main heading -->
+              <h1>Tonic</h1>
+            </div>
+            <div class="experience">
+              <!-- Card Frame  -->
+              <ul>
+                <li class="a">CANOPY</li>
+                <li class="b">Back End Dev</li>
+                <li class="c">2015</li>
+              </ul>
+            </div>
+            <div class="parga">
+              <!-- Card Des  -->
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="language">
+              <!-- Card Language  -->
+              <ul>
+                <li class="a l-a">html</li>
+                <li class="b">css</li>
+                <li class="c">javascript</li>
```

Comment:
- [ ] [optional] Kindly work more on this to ensure that your design is close to figma template design. 😄 

## Your design 

![crr20](https://user-images.githubusercontent.com/103386183/198845194-6fcb56f4-af4f-4e9d-bb9c-8925c7d60b20.png)

## Figma design

![crr21](https://user-images.githubusercontent.com/103386183/198845209-4fe0bf11-c918-49c4-ad79-6df94f31eb76.png)